ledger/accountdb.go

```diff
@@ -1131,3 +1131,205 @@ func (iterator *encodedAccountsBatchIter) Close() {
 		iterator.rows = nil
 	}
 }
+
+// orderedAccountsIter allows us to iterate over the accounts addresses in the order of the account hashes.
+type orderedAccountsIter struct {
+	step         int
+	rows         *sql.Rows
+	tx           *sql.Tx
+	accountCount int
+	accountData  bool
```

Comment:
can we rename "accountData" to something more descriptive like "includeAccountData" or whatever the appropriate field name would be?

Reply:
done.